LICENSE-3rdparty.csv

```diff
@@ -1,1408 +1,1408 @@
+core,github.com/lxn/win,BSD-3-Clause,The win authors
+core,github.com/magiconair/properties,BSD-2-Clause,Frank Schroeder
+core,github.com/mailru/easyjson,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/bootstrap,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/buffer,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/easyjson,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/gen,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/jlexer,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/jwriter,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/parser,MIT,Mail.Ru Group
+core,github.com/mattn/go-colorable,MIT,Yasuhiro Matsumoto
+core,github.com/mattn/go-isatty,MIT,Yasuhiro MATSUMOTO <mattn.jp@gmail.com>
+core,github.com/mattn/go-runewidth,MIT,Yasuhiro Matsumoto
+core,github.com/matttproud/golang_protobuf_extensions/pbutil,Apache-2.0,UNKNOWN
+core,github.com/mdlayher/netlink,MIT,Matt Layher
+core,github.com/mdlayher/netlink/nlenc,MIT,Matt Layher
+core,github.com/mdlayher/socket,MIT,Matt Layher
+core,github.com/mholt/archiver/v3,MIT,Matthew Holt
+core,github.com/mitchellh/copystructure,MIT,Mitchell Hashimoto
+core,github.com/mitchellh/go-homedir,MIT,Mitchell Hashimoto
+core,github.com/mitchellh/mapstructure,MIT,Mitchell Hashimoto
+core,github.com/mitchellh/reflectwalk,MIT,Mitchell Hashimoto
+core,github.com/mkrautz/goar,BSD-3-Clause,Mikkel Krautz
+core,github.com/moby/sys/mountinfo,Apache-2.0,UNKNOWN
+core,github.com/modern-go/concurrent,Apache-2.0,UNKNOWN
+core,github.com/modern-go/reflect2,Apache-2.0,UNKNOWN
+core,github.com/munnerz/goautoneg,BSD-3-Clause,Open Knowledge Foundation Ltd
+core,github.com/mxk/go-flowrate/flowrate,BSD-3-Clause,The Go-FlowRate Authors
+core,github.com/nwaples/rardecode,BSD-2-Clause,Nicholas Waples
+core,github.com/olekukonko/tablewriter,MIT,Oleku Konko
+core,github.com/opencontainers/go-digest,Apache-2.0,"Docker, Inc | OCI Contributors"
+core,github.com/opencontainers/image-spec/identity,Apache-2.0,The Linux Foundation
+core,github.com/opencontainers/image-spec/specs-go,Apache-2.0,The Linux Foundation
+core,github.com/opencontainers/image-spec/specs-go/v1,Apache-2.0,The Linux Foundation
+core,github.com/opencontainers/runc/libcontainer/cgroups,Apache-2.0,"Docker, Inc"
+core,github.com/opencontainers/runc/libcontainer/cgroups/fscommon,Apache-2.0,"Docker, Inc"
+core,github.com/opencontainers/runc/libcontainer/configs,Apache-2.0,"Docker, Inc"
+core,github.com/opencontainers/runc/libcontainer/devices,Apache-2.0,"Docker, Inc"
+core,github.com/opencontainers/runc/libcontainer/system,Apache-2.0,"Docker, Inc"
+core,github.com/opencontainers/runc/libcontainer/user,Apache-2.0,"Docker, Inc"
+core,github.com/opencontainers/runtime-spec/specs-go,Apache-2.0,The Linux Foundation
+core,github.com/opencontainers/selinux/go-selinux,Apache-2.0,UNKNOWN
+core,github.com/opencontainers/selinux/go-selinux/label,Apache-2.0,UNKNOWN
+core,github.com/opencontainers/selinux/pkg/pwalk,Apache-2.0,UNKNOWN
```

Comment:
497 - 499 ... https://github.com/opencontainers/selinux/blob/main/MAINTAINERS

don't know if we're giving definitive attribution if not listened as a notice or authors.

Reply:
Maybe 'the OpenContainers Community'?